server/tests/testBooking.js

```diff
@@ -307,7 +307,7 @@ describe('Test case: Booking endpoint /api/v1/bookings', () => {
     describe('Base case: Users should get a specific booking', () => {
         it('Should return 404. Booking not found', (done) => {
             request(app)
-                .get('/api/v1/bookings/4')
+                .get('/api/v2/bookings/4')
```

Comment:
This is good work. In the next OR, you can store the API URL in a variable so that you won't have to edit in in multiple places

Reply:
Also, ensure that you fill in every part of the PR description. Only the `questions` segment can be empty and the `screenshots` part in some cases